includes/definitions/discovery/bdcom.yaml

```diff
@@ -1,13 +1,127 @@
-mib: BDCOM-PROCESS-MIB
+mib: NMS-CHASSIS:NMS-FAN-TRAP:NMS-POWER-MIB:NMS-GPON-MIB:NMS-EPON-OLT-PON
 modules:
     mempools:
         data:
             -
-                used: BDCOM-MEMORY-POOL-MIB::bdcomMemoryPoolUsed
-                free: BDCOM-MEMORY-POOL-MIB::bdcomMemoryPoolFree
+                percent_used: NMS-CHASSIS::nmscardMEMUtilization
+                descr: 'Memory Usage'
+    os:
+        hardware:
+            - NMS-CHASSIS::nmscardDescr.0
+            - NMS-CHASSIS::nmscardDescr.1
+        version:
+            - NMS-CHASSIS::nmscardSwVersion.0
+            - NMS-CHASSIS::nmscardSwVersion.1
+        serial:
+            - NMS-CHASSIS::nmscardSerial.0
+            - NMS-CHASSIS::nmscardSerial.1
+    pre-cache:
+        data:
+            -
+                oid:
+                 - gponOltPonPortPortIndex
+                 - ponIfIndex
+                 - gponOltPonPortPortActiveOnuNum
+                 - gponOltPonPortPortInactiveOnuNum
     processors:
         data:
             -
-                oid: bdpmCPUTotal5min
-                num_oid: '.1.3.6.1.4.1.3320.9.109.1.1.1.1.5.{{ $index }}'
-                index: 0
+                oid: nmscardCPUUtilization
```

Comment:
According the the tests, it is using HOST-RESOURCES-MIB instead of this...  Maybe a mib failure?

Reply:
Seems better now with mibs added. But still wondering if the bdpmCPUTotal5min should be kept aside of nmscardCPUUtilization or replaced. 
@hrtrd Are you sure nmscardCPUUtilization works on any device/version ? wouldn't it make sense to keep both in the yaml file?